### ISSUE TITLE
📐 Theory — 2026-04-05 (28 papers)

### ISSUE THREAD

**Issue by DaehyeopKim** [DaehyeopKim/DaehyeopKim.github.io#57]
### 🏆 Oral

**1. [Generalized Linear Mode Connectivity for Transformers](https://openreview.net/forum?id=KurYdcCbjv)**
  *Alexander Theus, Alessandro Cabodi, Sotiris Anagnostidis et al.*
  📍 NeurIPS 2025 oral
  > Understanding the geometry of neural network loss landscapes is a central question in deep learning, with implications for generalization and optimization. A striking phenomenon is $\textit{linear ...

**2. [Implicit Regularization for Tubal Tensor Factorizations via Gradient Descent](https://openreview.net/forum?id=2GmXJnyNM4)**
  *Santhosh Karnik, Anna Veselovska, Mark Iwen et al.*
  📍 ICML 2025 oral
  > We provide a rigorous analysis of implicit regularization in an overparametrized tensor factorization problem beyond the lazy training regime. For matrix factorization problems, this phenomenon has...

**3. [Scaling Laws and Spectra of Shallow Neural Networks in the Feature Learning Regime](https://openreview.net/forum?id=Q3yLIIkt7z)**
  *Leonardo Defilippis, Yizhou Xu, Julius Girardin et al.*
  📍 ICLR 2026
  > Neural scaling laws underlie many of the recent advances in deep learning, yet their theoretical understanding remains largely confined to linear models. In this work, we present a systematic analy...

**4. [High-dimensional Analysis of Synthetic Data Selection](https://openreview.net/forum?id=Y54P2BBPPh)**
  *Parham Rezaei, Filip Kovačević, Francesco Locatello et al.*
  📍 ICLR 2026
  > Despite the progress in the development of generative models, their usefulness in creating synthetic data that improve prediction performance of classifiers has been put into question. Besides heur...

**5. [The Polar Express: Optimal Matrix Sign Methods and their Application to the Muon Algorithm](https://openreview.net/forum?id=yRtgZ1K8hO)**
  *Noah Amsel, David Persson, Christopher Musco et al.*
  📍 ICLR 2026
  > Computing the polar decomposition and the related matrix sign function has been a well-studied problem in numerical analysis for decades. Recently, it has emerged as an important subroutine  within...

**6. [Difficult Examples Hurt Unsupervised Contrastive Learning: A Theoretical Perspective](https://openreview.net/forum?id=5LMdnUdAoy)**
  *Yi-Ge Zhang, Jingyi Cui, Qiran Li et al.*
  📍 ICLR 2026
  > Unsupervised contrastive learning has shown significant performance improvements in recent years, often approaching or even rivaling supervised learning in various tasks. However, its learning mech...

**7. [Poincaré Gradient Descent: A Projection-Based Riemannian Method on the Poincaré Ball](https://openreview.net/forum?id=aH90wXwnyn)**
  *Chengyang Liu, Ouyang Shangke, Michael Ng et al.*
  📍 AAAI 2026
  > We propose Poincaré Gradient Descent (PGD), a first-order optimization method on the Poincaré ball. PGD replaces the exponential map with a projection-based retraction that is first-order equivalen...

**8. [MuSLR: Multimodal Symbolic Logical Reasoning](https://openreview.net/forum?id=MklzHxZPjx)**
  *Jundong Xu, Hao Fei, Yuhui Zhang et al.*
  📍 AAAI 2026
  > Multimodal symbolic logical reasoning, which aims to deduce new facts from multimodal input via formal logic, is critical in high-stakes applications such as autonomous driving and medical diagnosi...

### ⭐ Spotlight

**9. [Instance-Optimality for Private KL Distribution Estimation](https://openreview.net/forum?id=K0FbK2GOGj)**
  *Jiayuan Ye, Vitaly Feldman, Kunal Talwar*
  📍 NeurIPS 2025
  > We study the fundamental problem of estimating an unknown discrete distribution $p$ over $d$ symbols, given $n$ i.i.d. samples from the distribution. We are interested in minimizing the KL divergen...

**10. [Precise Asymptotics and Refined Regret of Variance-Aware UCB](https://openreview.net/forum?id=A0M3apV5zI)**
  *Yingying Fan, Yuxuan Han, Jinchi Lv et al.*
  📍 NeurIPS 2025
  > In this paper, we study the behavior of the Upper Confidence Bound-Variance (UCB-V) algorithm for the Multi-Armed Bandit (MAB) problems, a variant of the canonical Upper Confidence Bound (UCB) algo...

**11. [Conservative classifiers do consistently well with improving agents: characterizing statistical and online learning](https://openreview.net/forum?id=Zc5Ntjt5Ul)**
  *Dravyansh Sharma, Alec Sun*
  📍 NeurIPS 2025
  > Machine learning is now ubiquitous in societal decision-making, for example in evaluating job candidates or loan applications, and it is increasingly important to take into account how classified a...

**12. [Robust learning of halfspaces under log-concave marginals](https://openreview.net/forum?id=ZrCQGVpQrl)**
  *Jane Lange, Arsen Vasilyan*
  📍 NeurIPS 2025
  > We say that a classifier is $\text{\emph{adversarially robust}}$ to perturbations of norm $r$ if,
with high probability over a point $x$ drawn from the input distribution, there is no point within ...

**13. [Implicit Bias of Spectral Descent and Muon on Multiclass Separable Data](https://openreview.net/forum?id=Zn2ajV1kTQ)**
  *Chen Fan, Mark Schmidt, Christos Thrampoulidis*
  📍 NeurIPS 2025
  > Different gradient-based methods for optimizing overparameterized models can all achieve zero training error yet converge to distinctly different solutions inducing different generalization propert...

**14. [The Generative Leap: Tight Sample Complexity for Efficiently Learning Gaussian Multi-Index Models](https://openreview.net/forum?id=5X6PL4906S)**
  *Alex Damian, Jason D. Lee, Joan Bruna*
  📍 NeurIPS 2025
  > In this work we consider generic Gaussian Multi-index models, in which the labels only depend on the (Gaussian) $d$-dimensional inputs through their projection onto a low-dimensional $r = O_d(1)$ s...

**15. [Solving Neural Min-Max Games: The Role of Architecture, Initialization & Dynamics](https://openreview.net/forum?id=5xdbWUdM87)**
  *Deep Patel, Emmanouil-Vasileios Vlatakis-Gkaragkounis*
  📍 NeurIPS 2025
  > Many emerging applications—such as adversarial training, AI alignment, and robust optimization—can be framed as zero-sum games between neural nets, with von Neumann–Nash equilibria (NE)  capturing ...

**16. [Regularized least squares learning with heavy-tailed noise is minimax optimal](https://openreview.net/forum?id=TjQP5hc3WC)**
  *Mattes Mollenhauer, Nicole Mücke, Dimitri Meunier et al.*
  📍 NeurIPS 2025
  > This paper examines the performance of ridge regression in reproducing kernel Hilbert spaces 
in the presence of noise that exhibits a finite number of higher moments. We establish excess
risk boun...

**17. [On Feasible Rewards in Multi-Agent Inverse Reinforcement Learning](https://openreview.net/forum?id=qu6mRbSnUs)**
  *Till Freihaut, Giorgia Ramponi*
  📍 NeurIPS 2025
  > Multi-agent inverse reinforcement learning (MAIRL) aims to recover agent reward functions from expert demonstrations. We characterize the feasible reward set in Markov games, identifying all reward...

**18. [Stochastic Optimization in Semi-Discrete Optimal Transport: Convergence Analysis and Minimax Rate](https://openreview.net/forum?id=fGfl6dqQVf)**
  *Ferdinand Genans, Antoine Godichon-Baggioni, François-Xavier Vialard et al.*
  📍 NeurIPS 2025
  > We investigate the semi-discrete Optimal Transport (OT) problem, where a continuous source measure $\mu$ is transported to a discrete target measure $\nu$, with particular attention to the OT map a...

### 📌 Poster

**19. [Tight analyses of first-order methods with error feedback](https://openreview.net/forum?id=hlPk6Hi43e)**
  *Daniel Berg Thomsen, Adrien Taylor, Aymeric Dieuleveut*
  📍 NeurIPS 2025 poster
  > Communication between agents often constitutes a major computational bottleneck in distributed learning. One of the most common mitigation strategies is to compress the information exchanged, there...

**20. [Towards Generalizable 3D Human Pose Estimation via Ensembles on Flat Loss Landscapes](https://openreview.net/forum?id=nZ4mFzCZIx)**
  *Jumin Han, Jun-Hee Kim, Seong-Whan Lee*
  📍 NeurIPS 2025 poster
  > 3D Human Pose Estimation (HPE) is a fundamental task in the computer vision. Generalization in 3D HPE task is crucial due to the need for robustness across diverse environments and datasets. Existi...

**21. [A geometric framework for momentum-based optimizers for low-rank training](https://openreview.net/forum?id=cCefuzQrjK)**
  *Steffen Schotthöfer, Timon Klein, Jonas Kusch*
  📍 NeurIPS 2025 poster
  > Low-rank pre-training and fine-tuning have recently emerged as promising techniques for reducing the computational and storage costs of large neural networks. Training low-rank parameterizations ty...

**22. [Iterative Missing Data Imputation with Model Form Adaptation and Non-Missing Feature Supervision](https://openreview.net/forum?id=L84DdFuvwV)**
  *Hao Wang, zhengnan li, Zhichao Chen et al.*
  📍 NeurIPS 2025 poster
  > Iterative imputation is a prevalent method for missing data imputation, where each feature is imputed iteratively by treating it as a target variable estimated from all other features. However, ite...

**23. [Exploring Landscapes for Better Minima along Valleys](https://openreview.net/forum?id=XxRKqFsvoK)**
  *Tong Zhao, Jiacheng Li, Yuanchang Zhou et al.*
  📍 NeurIPS 2025 poster
  > Finding lower and better-generalizing minima is crucial for deep learning. However, most existing optimizers stop searching the parameter space once they reach a local minimum. Given the complex ge...

**24. [How Compositional Generalization and Creativity Improve as Diffusion Models are Trained](https://openreview.net/forum?id=1OUEnfusEd)**
  *Alessandro Favero, Antonio Sclocchi, Francesco Cagnetta et al.*
  📍 ICML 2025 poster
  > Natural data is often organized as a hierarchical composition of features. How many samples do generative models need in order to learn the composition rules, so as to produce a combinatorially lar...

**25. [Rethinking Benign Overfitting in Two-Layer Neural Networks](https://openreview.net/forum?id=Uc0dTE2Wox)**
  *Ruichen Xu, Kexin Chen*
  📍 ICML 2025 poster
  > Recent theoretical studies (Kou et al., 2023; Cao et al., 2022) revealed a sharp phase transition from benign to harmful overfitting when the
noise-to-feature ratio exceeds a threshold—a situation ...

**26. [The Importance of Being Lazy: Scaling Limits of Continual Learning](https://openreview.net/forum?id=edhBkkYS8R)**
  *Jacopo Graldi, Alessandro Breccia, Giulia Lanzillotta et al.*
  📍 ICML 2025 poster
  > Despite recent efforts, neural networks still struggle to learn in non-stationary environments, and our understanding of catastrophic forgetting (CF) is far from complete.
In this work, we perform ...

**27. [Generalization of noisy SGD in unbounded non-convex settings](https://openreview.net/forum?id=Au9rfI6Fjd)**
  *Leello Tadesse Dadi, Volkan Cevher*
  📍 ICML 2025 poster
  > We study generalization of iterative noisy gradient schemes on smooth non-convex losses. Formally, we establish time-independent information theoretic generalization bounds for Stochastic Gradient ...

**28. [High Probability Bounds for Non-Convex Stochastic Optimization with Momentum](https://openreview.net/forum?id=KirKWFPYJA)**
  *Shaojie Li, Pengwei Tang, Bowei Zhu et al.*
  📍 ICLR 2026 Poster
  > Stochastic gradient descent with momentum (SGDM) is widely used in machine learning, yet high-probability learning bounds for SGDM in non-convex settings remain scarce. In this paper, we provide hi...
